test/interpreters/test_subprocess_interpreter.py

```diff
@@ -0,0 +1,118 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+from pathlib import Path
+
+import pytest
+
+from camel.interpreters import InterpreterError, SubprocessInterpreter
+
+
+@pytest.fixture
+def subprocess_interpreter():
+    return SubprocessInterpreter(
+        require_confirm=False,
+        print_stdout=True,
+        print_stderr=True,
+    )
+
+
+def test_run_python_code(subprocess_interpreter):
+    python_code = """
+def add(a, b):
+    return a + b
+
+result = add(10, 20)
+print(result)
+"""
+    result = subprocess_interpreter.run(python_code, "python")
+    assert "stdout: 30\n" in result
+    assert "stderr: " in result
```

Comment:
This `assert` does not give much helpful informaiton about the test

Reply:
Because this test prints nothing to stderr, this assertion just makes sure the output follows the format.